app/services/stepfunctions/RegularContributionsClient.scala

```diff
@@ -54,6 +54,22 @@ object RegularContributionsClient {
 
 case class StatusResponse(status: Status, trackingUri: String, failureReason: Option[CheckoutFailureReason] = None)
 
+case class StatusResponseWithOptionalGuestAccountToken(
```

Comment:
I think it would be simpler to modify the existing case class:

```case class StatusResponse(status: Status, trackingUri: String, failureReason: Option[CheckoutFailureReason] = None, guestAccountCreationToken: Option[String] = None)```

Reply:
Yeah you're probably right. Will change